Library/Homebrew/cask/lib/hbc/cli/search.rb

```diff
@@ -15,8 +19,18 @@ def self.extract_regexp(string)
       end
 
       def self.search_remote(query)
-        matches = GitHub.search_code(user: "caskroom", path: "Casks",
-                                     filename: query, extension: "rb")
+        matches = begin GitHub.search_code(
+          user: "caskroom",
+          path: "Casks",
+          filename: query,
+          extension: "rb"
+        )
+        rescue Exception => e
```

Comment:
I changed it to `StandardError`. The error raised is [this](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/utils/github.rb#L220-L226), can it be made more specific?